integrations/perfecto_flutter_sample_project/README.md

```diff
@@ -0,0 +1,5 @@
+# Integrating Codemagic with Perfecto for Flutter apps
+
+## Getting Started
+
+Refer to the documentation here for a thorough explanation on how to integrate Codemagic with Perfecto.
```

Comment:
Missing link?

Reply:
yes, i need to add this sample project to the docs' PR. After merging the docs PR, i'll add a link.